src/components/webView/iframe.native.js

```diff
@@ -66,19 +67,22 @@ export const createIframe = (src, title, backToWallet = false, backToRoute = 'Ho
       },
     }
 
-    const NavigationBar = navigate => (
-      <Appbar.Header dark style={navBarStyles.wrapper}>
-        <View style={{ width: 48 }} />
-        <Appbar.Content />
-        <Section.Text color="white" fontWeight="medium" style={navBarStyles.title} testID="rewards_header">
-          {title}
-        </Section.Text>
-        <Appbar.Content />
-        <TouchableOpacity onPress={() => navigate(backToRoute)} style={navBarStyles.walletIcon}>
-          <Icon name="wallet" size={36} color="white" />
-        </TouchableOpacity>
-      </Appbar.Header>
-    )
+    const NavigationBar = navigate => {
```

Comment:
probably this could to be reused ?
https://github.com/GoodDollar/GoodDAPP/pull/2197/files#diff-8fa47377fc065bea8612274545359352R88

Reply:
done